### DISCUSSION TITLE
Question about Numbering Systems....

### DISCUSSION THREAD

**Discussion by reknoZ** [moment/luxon#1152] [Q&A] [answered]
Hi there,

I just saw some code that uses Luxon and it use a numbering system ("beng"). I've found other references to "arab" and "latn" (I guess that's latin?).

Can someone please explain what "beng" is and/or give a full, detailed list of all possible values for this?

Thanks.

**Comment by icambron** [comment, marked as answer]
See here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat

> Numbering system. Possible values include: "arab", "arabext", "bali", "beng", "deva", "fullwide", "gujr", "guru", "hanidec", "khmr", "knda", "laoo", "latn", "limb", "mlym", "mong", "mymr", "orya", "tamldec", "telu", "thai", "tibt".

**Reply by reknoZ** [reply]
Thank you very much for this resource/answer. Now, does anyone know what beng is? or deva? or many of the others?

**Reply by icambron** [reply]
Beng is short for Bengali -- so presumably the numbers used in Bengladesh.

Here is what the different abbreviations stand for:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/numberingSystem, though note that not all those options can be used with DateTimes
